sawmill_api/migrations/V20250618195438__api_settings.sql

```diff
@@ -0,0 +1,8 @@
+CREATE SCHEMA api;
+
+CREATE TABLE api.settings(
+    section TEXT NOT NULL,
+    name TEXT NOT NULL,
+    value TEXT NOT NULL,
```

Comment:
I know, text, right!? Well, I couldn't get something like JSONB to work as expected. Like, I _assumed_ casting in the SQL _to_ JSONB would be just fine, but nope! So, screw it, make it text. Pydantic will coerce it when I pull it out of the database.